beamer/tests/agent/test_request.py

```diff
@@ -87,13 +87,14 @@ def test_challenge_own_claim(config, request_manager, token):
     agent_address = to_checksum_address(agent.address)
     claim_stake = request_manager.claimStake()
     request = Request(
-        RequestId(100),
+        RequestId(31 * b"" + b"1"),
```

Comment:
Why not just write here `b"1"`?
Or did you really think to write `b"\0"` which would be byte array of 31 zero bytes followed by a byte value 49?

Reply:
In some occasions the tests didnt work because of that. But in this case the byte creation doesnt make a difference. I think it fails when we compare a contract keccak to a locally created one.